setup.py

```diff
@@ -11,7 +11,7 @@
     author       = 'Dmitry Blotsky',
```

Comment:
BTW, feel free to add your name here too!

Reply:
Awesome! Thanks!